deployment/mmw_stack.py

```diff
@@ -72,7 +72,7 @@ def main():
                                     parents=[common_parser])
     mmw_ami.add_argument('--machine-type', type=str,
                          choices=['mmw-app', 'mmw-tiler', 'mmw-worker',
-                                  'mmw-monitoring'],
+                                  'mmw-monitoring', 'mmw-app,mmw-worker'],
```

Comment:
Consider making this argument accept a comma delimited list of any machine type. (What if we want to build the tiler in parallel as well?)